### ISSUE TITLE
Replace hg up null workaround with the new workaround from https://bugzilla.mozilla.org/show_bug.cgi?id=1720302

### ISSUE THREAD

**Issue by marco-c** [mozilla/libmozevent#69]
The new workaround from https://bugzilla.mozilla.org/show_bug.cgi?id=1720302 is much faster than `hg up null` we introduced in #68.

The workaround script lives at https://gist.github.com/zzzeid/46f7d8cc794f9fccf999bb8356f981be.

**Comment by abpostelnicu** [comment]
@marco-c if the current solution works, why don't we wait for the final patch?

**Comment by marco-c** [comment]
Yeah, maybe better to wait at this point. We can decide to use this new workaround in case we see the bot being too slow because of `hg up null`.

**Comment by marco-c** [comment]
The repo was fixed, so we can actually remove the workaround.
